pdf-aam/index.html

```diff
@@ -191,33 +141,40 @@ <h2>General rules</h2>
 
         <section id="mapping_role_table">
           <h3>Role Mapping Tables</h3>
-          <h4 id="role-map-example"><code>example</code></h4>
+          <h4 id="role-map-example"><code>Document Logical document within a parent document</code></h4>
```

Comment:
```suggestion
          <h4 id="role-map-example"><code>document</code> Logical document within a parent document</h4>
```

Beyond the formatting diff, are there multiple types of documents you're trying to represent?

Reply:
For the most part, you'll want the output to use a similar source/structure as these tables in HTML-AAM or DPUB-AAM, which will get you a similar final ReSpec rendering to this:

![ReSpec rending of the HTML a:link mapping, which cross-references ARIA's link role](https://github.com/user-attachments/assets/095fc94e-ad43-4655-8470-7fcf522c8539)

Reply:
And then only in situations where there is no other mapping to through to would you define the system-specific API properties... Here's HTML-AAM's `abbr` as an example:

![ReSpec rendering of the HTML-AAM abbr table](https://github.com/user-attachments/assets/b9a9df45-4fe2-437b-9aeb-d573073889e6)

Reply:
And in that last case, the computed role would be something like `pdf-foo`

Reply:
Yes, at least one other entry, "Child of Structure Tree root", that we have identified as of now.

Reply:
There are probably places you'll want to point to the DPUB spec, too.

Reply:
I think we will need to apply something along these lines for attribute entries, specifically the last entry in our initial import. @PaulRayius